pyproject.toml

```diff
@@ -67,7 +67,8 @@ googleapis-common-protos = ">=1.53.0"
 lz4 = ">=3.1.3"
 protobuf = "~=3.20.0"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
```

Comment:
This is the poetry-recommended way of declaring dev dependencies. I think since 1.2 they're considered equivalent, but they recommend moving over. Check this section out https://python-poetry.org/docs/managing-dependencies/#dependency-groups and scroll to `A note about defining a dev dependencies group`.